templates/services/nextcloud/nextcloud_yml.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+#create nextcloud.yml
+
+mkdir -p /srv/nextcloud
+
+{
+ echo "services:"
+ echo "  nextcloud:"
+ echo "    image: nextcloud"
+ echo "    ports:"
+ echo "      - \"8081\""
```

Comment:
You shouldn't be changing the port in the yml file.

What was incorrect was the port being passed to the `check_tor` function in `services`.

Refer to the comment on `services.sh`.